.travis.yml

```diff
@@ -44,9 +52,10 @@ install:
 
 script:
 # Build the package, its tests, and its docs and run the tests
-- stack --no-terminal build --fast
+- stack --no-terminal build --fast --copy-bins
```

Comment:
Are we copying the binaries built from here into the Dockerfile? If so, we shouldn't be using --fast to build the production binaries.

Reply:
Just confirming before I update the code... Would `stack --no-terminal build --copy-bins` be enough or is there a better command I should use here?

Reply:
That'll do it.